v2/exchange_info_service.go

```diff
@@ -19,7 +19,15 @@ func (s *ExchangeInfoService) Symbol(symbol string) *ExchangeInfoService {
 }
 
 // Symbols set symbol
-func (s *ExchangeInfoService) Symbols(symbols ...string) *ExchangeInfoService {
+func (s *ExchangeInfoService) Symbols(pairs ...string) *ExchangeInfoService {
+	symbols := "["
+	for i, pair := range pairs {
+		if i != 0 {
+			symbols += ","
+		}
+		symbols += "\"" + pair + "\""
+	}
+	symbols += "]"
```

Comment:
Just formatted your code in a cleaner way. You also need to import `strings` package.

```suggestion
	if len(pairs) == 0 {
	        s.symbols = "[]"
	} else {
	        s.symbols = "[\"" + strings.Join(pairs, "\",\"") + "\"]"
	}
	return s
```

Reply:
@Cih2001 I believe the tests should be also updated a bit.